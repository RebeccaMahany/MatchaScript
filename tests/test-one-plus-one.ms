print("I don't know much about algebra but I know that one plus one equals");
print(1+1);