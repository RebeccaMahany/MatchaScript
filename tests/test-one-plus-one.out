I don't know much about algebra but I know that one plus one equals
2
